File: Unidad 3/Java/Programa13U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa13u3;

import java.util.Scanner;


public class Programa13U3 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
          int c, num, contadorp=1,contadorn=1; 

        for (c=1; c<=100; c++){
            System.out.println("Escribe el numero: ");
            num = sc.nextInt();
            
            if(num > 0){
                System.out.println("Son: "+ contadorp+" numeros positivos");
                contadorp=contadorp+1;  
            }else{
                System.out.println("Son: "+ contadorn+" numeros negativos");
                contadorn=contadorn+1;
            }
    }
}
    }

File: Unidad 3/C#/Programa13U3.cs
// See https://aka.ms/new-console-template for more information

using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class Programa13U3
    {
        static void Main(string[] args)
        {
            int c, num, contadorp = 1, contadorn = 1;

            for (c = 1; c <= 100; c++)
            {
                Console.WriteLine("Escribe el numero: ");
                num = Convert.ToInt32(Console.ReadLine());

                if (num > 0)
                {
                    Console.WriteLine("Son: " + contadorp + " numeros positivos");
                    contadorp = contadorp + 1;
                }
                else
                {
                    Console.WriteLine("Son: " + contadorn + " numeros negativos");
                    contadorn = contadorn + 1;
                }
            }
        }
    }
}